src/Lock.php

```diff
@@ -252,6 +270,15 @@ public static function showForItem(CommonDBTM $item)
         ]);
         $header |= $results['header'];
 
+        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
```

Comment:
Should not this message be hidden if itemtype is not computer?

Reply:
nop, because ```Item_SoftwareVersion``` and ```Item_SoftwareLicense``` are displayed (if available) regardless of the itemtype